src/utils/vc/credentials.js

```diff
@@ -250,17 +284,15 @@ export async function verifyCredential(credential, {
  *   `@context` (if it is not present, it's added to the context list).
  * @return {Promise<object>} The signed credential object.
  */
-export async function issueCredential(keyDoc, credential, compactProof = true, documentLoader = null, purpose = null, expansionMap = null, issuerObject = null, addSuiteContext = false, useProofValue = false) {
-  // Get suite from keyDoc parameter
-  const suite = getSuiteFromKeyDoc(keyDoc, useProofValue);
-  if (!suite.verificationMethod) {
-    throw new TypeError('"suite.verificationMethod" property is required.');
-  }
+export async function issueCredential(keyDoc, credential, compactProof = true, documentLoader = null, purpose = null, expansionMap = null, issuerObject = null, addSuiteContext = true, type = false) {
+  const useProofValue = type === 'proofValue';
+  const useJWT = type === 'jwt';
 
   // Clone the credential object to prevent mutation
   const issuerId = credential.issuer || keyDoc.controller;
   const cred = {
     ...credential,
+    // issued: credential.issuanceDate,
```

Comment:
commented code?

Reply:
need to see if it makes sense to set this field in the SDK or if called should decide